testdata/kata/subscription.yaml

```diff
@@ -1,13 +1,12 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: kata-operator
-  namespace: sandboxed-containers-operator-system
+  name: sandboxed-containers-operator
+  namespace: openshift-sandboxed-containers-operator
 spec:
-  channel: <%= cb.cluster_ver %>
+  channel: "preview-1.1"
```

Comment:
I guess that the channel here should be replaced by new channel.
Perhaps possible to leave it as "channel: <channel>" instead of the actual value

Reply:
it will brake our CI and manual tests, i'll take in account in the next PR and i'll make it parameterized, for now this channel is supported for 4.8/9, our official releases

Reply:
@rrasouli